### PR TITLE
Migrate TestAuthAPI from integration-cli to integration

### PR DESCRIPTION
**Pull request by yongtang**
This fix migrates TestAuthAPI from integration-cli to integration

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

File: integration/system/login_test.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/integration/util/request"
+	"github.com/docker/docker/integration/util/requirement"
+	"github.com/gotestyourself/gotestyourself/skip"
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/net/context"
+)
+
+// Test case for GitHub 22244
+func TestAuthAPI(t *testing.T) {
```
Comment (dnephin) [inline integration/system/login_test.go]: `TestLoginFailsWithBadCredentials`

**Review by dnephin — APPROVED** [review]
LGTM, just a nit on the test name

**Review by vdemeester — APPROVED** [review]
LGTM 🐯

**Comment by yongtang** [comment]
@dnephin Update. Thanks.

**Pull request merged**